src/components/Navbar/Navbar.js

```diff
@@ -21,22 +22,18 @@ export default function Navbar() {
     return (
         <nav className={`nav ${showBackgroundClass}`}>
             <div className="nav__content">
-                <Link to='/'>
+                <Link 
+                to={PATH.HOME}
+                >
```

Comment:
This element has only one attribute, so you can keep it in one line. Or at least add indentation